### DISCUSSION TITLE
Can the H265 encoded video stream be uploaded to mediamtx through RTSP and accessed through webRTC

### DISCUSSION THREAD

**Discussion by shanjunmei** [bluenviron/mediamtx#3365] [Questions]
### Question

Can the H265 encoded video stream be uploaded to mediamtx through RTSP and accessed through webRTC

**Comment by aler9** [comment]
Hello, yes if you re-encode the stream to H264 by following instructions in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#remuxing-re-encoding-compression

**Comment by aler9** [comment]
streaming H265 tracks with WebRTC has been implemented with #4003
